docs/guides/admin/docs/configuration/episode-id-roles.md

```diff
@@ -0,0 +1,21 @@
+Episode ID roles
+=================================
+
+If activated, users with a role like ROLE_EPISODE_<ID>_<ACTION> will have access to the episode with the given
+identifier, without this having to be specifically specified in the ACL attached to the episode.
```

Comment:
```suggestion
identifier, without this having to be explicitly stated in the ACL attached to the episode.
```

Replacing both instances of a repetition just because I can 😎